app/src/main/res/drawable/ic_circle_thumbs_up_activated.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Im using a `<selector>` for states and selector doesn't support tint. We can implement this StateListDrawable in Java/Kotlin if needed